exchange/multibid.go

```diff
@@ -0,0 +1,70 @@
+package exchange
+
+import (
+	"fmt"
+
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+const DefaultBidLimit = 1
+const MaxBidLimit = 9
+
+type ExtMultiBidMap map[string]*openrtb_ext.ExtMultiBid
+
+// Validate and add multiBid value
+func (mb *ExtMultiBidMap) Add(multiBid *openrtb_ext.ExtMultiBid) []error {
```

Comment:
Each line in this function where `errs` gets appended to isn't covered by tests

Reply:
added test cases under `TestExtMultiBidMap_Add()`